pkgs/development/python-modules/python-ldap-test/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, py4j }:
+
+buildPythonPackage rec {
+  pname = "python-ldap-test";
+  version = "0.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qh9x3lncaldnw79fgpqbayichs8pbz8abr6pxb5qxbs7zrnyrwf";
+  };
+
+  propagatedBuildInputs = [ py4j ];
+
+  # Tests needs java to be present in path
+  doCheck = false;
```

Comment:
would it be sufficient to add Java as checkInput and alter `$PATH` in the `checkPhase`?

I'm not entirely sure yet if we actually want to add `java` to the closure.

Reply:
@Ma27  I think that might work. Nonetheless, I checked the other python packages and I couldn't find any one using JAVA so i dropped the idea.

Reply:
true, but I don't know if there are that much other python packages that require java for any part of the build process :)

Reply:
Aside from using unit tests, I doubt we'll be able to actually test speaking to an LDAP server during a build :)

Reply:
So, Should we enable the tests or not?  :confused: